### ISSUE TITLE
"file" Option Broken in Config

### ISSUE THREAD

**Issue by AntumDeluge** [lunarmodules/LDoc#351]
Just curious why I'm getting the following output:
```
c:\games\minetest\mods\mineral\init.lua:34: no module() call found; no initial doc comment
c:\games\minetest\mods\mineral\items.lua:52: no module() call found; no initial doc comment
c:\games\minetest\mods\mineral\nodes.lua:27: no module() call found; no initial doc comment
c:\games\minetest\mods\mineral\ores.lua:46: no module() call found; no initial doc comment
```

Here are the settings in `config.ld`:
```lua
title = "Minerals for Minetest"
project = "minerals"
format = "markdown"
not_luadoc = true
boilerplate = false
icon = "icon.png"

file = {"settings.lua", "api.lua",}
```

I am invoking it like this:
```
ldoc -c "C:\Games\Minetest\mods\mineral\docs\config.ld" -d "C:\Games\Minetest\mods\mineral\docs" \
  -o "api" "C:\Games\Minetest\mods\mineral"
```

My understanding is that it should only be parsing the files `settings.lua` & `api.lua`.

**Comment by AntumDeluge** [comment]
Only appears to be an issue with `file` variable within the config. Setting file using command line parameter works fine.

**Comment by AntumDeluge** [comment]
I believe this may be partially due to the fact that the file argument is required in the command line. So `file` is never used from the config.

**Comment by AntumDeluge** [comment]
It turns out that my issue is that LDoc only recognizes `file` from the config if the `file` command line parameter is `.`. I created a pull request to fix this: #356

Related: #279
